server/service/src/pack_unit.rs

```diff
@@ -0,0 +1,176 @@
+use std::collections::HashMap;
+
+use repository::{
+    EqualFilter, PackUnitRow, PackUnitRowRepository, RepositoryError, StockLineFilter,
+    StockLineRepository, StockOnHandFilter, StockOnHandRepository, StorageConnection,
+};
+
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub struct PackUnit {
+    pub item_id: String,
+    pub most_used_pack_unit_id: String,
+    pub pack_units: Vec<PackUnitRow>,
```

Comment:
Is this `most_used_pack_unit_id`? 

Think a doc comment for the struct fields is helpful here, especially for `pack_units` and `most_used_pack_size_id`...

The `PackUnit` struct name is a bit confusing because it contains a `pack_units` field. Maybe `ItemPackUnit`

Reply:
Or PackUnitSummary or similar... can't think of a good name